src/components/Inputs/input.module.scss

```diff
@@ -11,6 +11,7 @@
         medium: $input-medium-size,
         large: $input-large-size,
     );
+    --border-color: #69717f;
```

Comment:
you can have a localized var if you'd like, but name it something input specific and put it above with the others on lines above. Also, no need to set a specific value here, as you can assign a var from the existing palettes. Recommend this:

--input-border-color: var(--grey-color-60);

Reply:
done updated